models/articles.models.js

```diff
@@ -31,4 +31,18 @@ function fetchAllArticles() {
         })
 }
 
-module.exports = { fetchArticleById, fetchAllArticles }
+function fetchAllCommentsByArticleId(id) {
+    return db.query(`
+        SELECT comment_id, votes, created_at, author, body, article_id
+        FROM comments
+        WHERE article_id=$1
+        ORDER BY created_at DESC`, [id])
+        .then(({rows}) => {
```

Comment:
Is it an error if there are no comments? You're querying the comments table, not the articles table. If I wrote a brand new article, how many comments would it have?